### ISSUE TITLE
About Sticky Nav

### ISSUE THREAD

**Issue by Israel-Beni** [Israel-Beni/HTML-Tags-Cheat-Sheet#2]
Hi @oscar-fernandez, I really appreciate your contribution. But there is a small issue. When the `sticky functionality` is on, the very bottom of the nav is hidden. Please check that. Thanks.

**Comment by oscar-fernandez** [comment]
Yeah, that will happen with smaller screens. I can work on making the whole page responsive.

**Comment by Israel-Beni** [comment]
> Yeah, that will happen with smaller screens. I can work on making the whole page responsive.

Ok. You can go ahead.

**Comment by Israel-Beni** [comment]
I was also thinking to extend the project to all the tags of HTML. As you can see this cheat sheet, right now, has only the Form tags.
